packages/rx_bloc_cli/mason_templates/bricks/lib_social_logins/__brick__/android/app/src/main/res/values/string.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
I guess the moustache condition needs to be added here

Reply:
This file will be generated only if social logins brick is enabled, so do moustache condition really needs to be added here?

Reply:
Ahh you are right.